### ISSUE TITLE
The return value from a custom beforeunload script dialog is ignored

### ISSUE THREAD

**Issue by danjag** [MicrosoftEdge/WebView2Feedback#628]
**Description**
The return value from a custom beforeunload script dialog is ignored

**Version**
SDK: 1.0.674-prerelease
Runtime: 88.0.692.0 dev
Framework: Win32
OS: Windows 10 1909

**Repro Steps**
If I disable the builtin script dialogs:

```
ICoreWebView2SettingsPtr Settings;
RETURN_IF_FAILED(WebView->get_Settings(&Settings));
RETURN_IF_FAILED(Settings->put_AreDefaultScriptDialogsEnabled(FALSE));
```
and show my own dialog for dialog kind COREWEBVIEW2_SCRIPT_DIALOG_KIND_BEFOREUNLOAD

It does not matter whether I call **ICoreWebView2ScriptDialogOpeningEventArgs::Accept** or not - the navigation continues anyway.

If I use the built in dialogs: 
`RETURN_IF_FAILED(Settings->put_AreDefaultScriptDialogsEnabled(TRUE));`
I can cancel the navigation as expected.

[AB#30555568](https://microsoft.visualstudio.com/90b2a23c-cab8-4e7c-90e7-a977f32c1f5d/_workitems/edit/30555568)

**Comment by champnic** [comment]
Thanks for reporting this, I've added it to our backlog to take a look at.

**Comment by monica-ch** [comment]
We have fixed the issue in Edge canary 115.0.1830.0+. Please validate and let us know if you have more questions.
